_posts/2017-10-06-DevopsDays-London.md

```diff
@@ -0,0 +1,217 @@
+---
+layout:     post
+title:      DevopsDays London 2017
+date:       2017-10-06
+summary:    A look at DevopsDays London 2017
+category:   Devops
+tags:       Devops
+author:     adam stacey 
+---
+Introduction
+============
+
+Core are the Tribe that works across Sky Betting and Gaming – we ensure
```

Comment:
The introduction of Core Tribe feels a bit out of place here, maybe move it further down and explain why it is important to the rest of the article? ie. you are attending and writing about the conference with this team in mind.